documentation/modules/ROOT/pages/development/engine.adoc

```diff
@@ -85,8 +85,9 @@ Here's an example of code that configures and runs an embedded xref:{link-mysql-
 [source,java,indent=0]
 ----
     // Define the configuration for the Debezium Engine with MySQL connector...
-    final Properties props = config.asProperties();
+    final Properties props = new Properties();
     props.setProperty("name", "engine");
+    props.setProperty("connector.class", "io.debezium.connector.mysql.MySqlConnector");
```

Comment:
this showed up in the break-down below, but was missing here, which is in the way of copy and paste of the entire snippet.